change/@fluentui-react-message-bar-0408d1da-e3d1-4d30-955d-7fa6120e573d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31749&commitId=b634fcf7fb19d6760015b28a011faf7962f4a187&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.basic - High Contrast.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.size.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.normal.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.selected.chromium.png|0| Removed|

</details>


  <details><summary><b>MessageBar</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MessageBar.Intents - High Contrast.default.chromium.png|16| Changed|
|MessageBar.Intents - RTL.default.chromium.png|11| Changed|
|MessageBar.Multiline Without Actions.default.chromium.png|11| Changed|
|MessageBar.Multiline.default.chromium.png|11| Changed|
|MessageBar.Intents - Dark Mode.default.chromium.png|7| Changed|
|MessageBar.Intents.default.chromium.png|11| Changed|
|MessageBar.Multiline No Actions.default.chromium.png|11| Changed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Shape.default.chromium.png|0| Added|
|SwatchPicker Converged.spacing.default.chromium.png|0| Added|
|SwatchPicker Converged.default - High Contrast.default.chromium.png|0| Removed|
|SwatchPicker Converged.default - RTL.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>